workflow/rocoto/gfs_tasks.py

```diff
@@ -868,6 +868,18 @@ def gempak(self):
 
         return task
 
+    def npoess(self):
+
+        deps = []
+        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}postanl'}
```

Comment:
`postanl` is not a metatask
```suggestion
        dep_dict = {'type': 'task', 'name': f'{self.cdump}postanl'}
```